css/selectors/checked.json

```
@@ -7,34 +7,44 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/:checked",
           "support": {
             "chrome": {
-              "version_added": "1"
+              "version_added": "1",
+              "notes": "On macOS, styling <code>&lt;option&rt;</code> elements has no effect."
             },
             "chrome_android": {
               "version_added": "18"
             },
             "edge": {
-              "version_added": "12"
+              "version_added": "12",
+              "notes": "On macOS, styling <code>&lt;option&rt;</code> elements has no effect."
             },
             "firefox": {
-              "version_added": "1"
+              "version_added": "1",
+              "notes": [
+                "Support for styling <code>&lt;option&rt;</code> elements was removed in Firefox 56.",
```

One last fix. Thank you!

```suggestion
                "From Firefox 56, <code>&lt;option&rt;</code> elements cannot be styled.",
```